modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="enabling-assisted-installer-service-on-bare-metal_{context}"]
+= Enabling assisted installer service on bare metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. {rh-rhacm-first} ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the {rh-rhacm} hub cluster.
+
+For distributed units (DUs), {rh-rhacm} supports {product-title} deployments, which run on a single bare metal host. The single node cluster acts as both a control plane and a worker node. {rh-rhacm} supports single node OpenShift deployments for {product-title} on top of a single bare metal Operator (BMO) DU.
+
+.Prerequisites
+
+* Install {product-title} {product-verstion} on a hub cluster.
```

Comment:
```suggestion
* Install {product-title} {product-version} on a hub cluster.
```

Reply:
`{product-version}` is rendering as `Branch Build` in the preview output. @kalexand-rh, do we need to call out product version in the prereqs?